code/game/mecha/equipment/tools/medical_tools.dm

```diff
@@ -488,7 +488,7 @@
 	return 1
 
 /obj/item/mecha_parts/mecha_equipment/medical/syringe_gun/proc/add_known_reagent(r_id,r_name)
-	if(!(r_id in known_reagents))
+	if(!(r_id in known_reagents) && (r_id in standard_medicines))

```

Comment:
This is probably a bit more of a balance change, might be preferable to just block the chems in [this list](https://github.com/ParadiseSS13/Paradise/blob/master/code/_globalvars/lists/reagents.dm#L50) instead.